### PR TITLE
chore: Update readme for GraphQL spec compability

### PR DESCRIPTION
**Pull request by calvincestari**
Trying to make it clearer for users to know which version of the GraphQL spec the client is compatible with.

### CODE REVIEW

**Comment by calvincestari** [comment]
Keeping this as a draft until we've all agreed on the messaging.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Because generated types are operation-specific, you can only access data that yo
 
 This library is designed with Android in mind, but you can use it in any Kotlin application, including KMP ([Kotlin Multi Platform](https://kotlinlang.org/docs/multiplatform.html)).
 
+Apollo Kotlin aims to comply with the the [Working Draft of the GraphQL specification](https://spec.graphql.org/draft/).
```
Comment (martinbonnin) [inline README.md:36]:
```suggestion
Apollo Kotlin aims to comply with the [Working Draft of the GraphQL specification](https://spec.graphql.org/draft/).
```

Comment (martinbonnin) [reply README.md]: Not judging, this is one of my signature move 🙈

**Review by martinbonnin — COMMENTED** [review]
Thanks!

**Pull request merged**